### PR TITLE
Translate  Header

### PR DESCRIPTION
**Pull request by fkanout**
By merging this PR, and #36 #37 #38 the Home page become entirely translated  🎉

### CODE REVIEW

File: src/components/LayoutHeader/Header.js [hunk 1]
```diff
@@ -122,22 +122,22 @@ const Header = ({location}: {location: Location}) => (
...
           <HeaderLink
             isActive={location.pathname.includes('/docs/')}
-            title="Docs"
+            title="مستندات"
             to="/docs/getting-started.html"
           />
           <HeaderLink
             isActive={location.pathname.includes('/tutorial/')}
-            title="Tutorial"
+            title="دورة تعليمية"
```
Comment (asantarissy) [inline src/components/LayoutHeader/Header.js]: الدليل التطبيقي

File: src/components/LayoutHeader/Header.js [hunk 2]
```diff
@@ -122,22 +122,22 @@ const Header = ({location}: {location: Location}) => (
           }}>
           <HeaderLink
             isActive={location.pathname.includes('/docs/')}
-            title="Docs"
+            title="مستندات"
```
Comment (asantarissy) [inline src/components/LayoutHeader/Header.js]: المستندات

File: src/components/LayoutHeader/Header.js [hunk 3]
```diff
@@ -122,22 +122,22 @@ const Header = ({location}: {location: Location}) => (
...
           <HeaderLink
             isActive={location.pathname.includes('/tutorial/')}
-            title="Tutorial"
+            title="دورة تعليمية"
             to="/tutorial/tutorial.html"
           />
           <HeaderLink
             isActive={location.pathname.includes('/blog')}
-            title="Blog"
+            title="مدونة"
```
Comment (asantarissy) [inline src/components/LayoutHeader/Header.js]: المدونة

File: src/components/LayoutHeader/Header.js [hunk 4]
```diff
@@ -122,22 +122,22 @@ const Header = ({location}: {location: Location}) => (
...
             to="/blog/"
           />
           <HeaderLink
             isActive={location.pathname.includes('/community/')}
-            title="Community"
+            title="مجتمع"
```
Comment (asantarissy) [inline src/components/LayoutHeader/Header.js]: مجتمع المطورين

File: src/components/LayoutHeader/Header.js [hunk 5]
```diff
@@ -179,7 +179,7 @@ const Header = ({location}: {location: Location}) => (
               },
             }}
             to="/versions">
-            v{version}
+            اصدار{version}
```
Comment (asantarissy) [inline src/components/LayoutHeader/Header.js]: Add space in-between

File: src/components/LayoutHeader/Header.js (hunk 1 above)
Comment (fkanout) [reply src/components/LayoutHeader/Header.js]: done

File: src/components/LayoutHeader/Header.js (hunk 2 above)
Comment (fkanout) [reply src/components/LayoutHeader/Header.js]: done

File: src/components/LayoutHeader/Header.js (hunk 3 above)
Comment (fkanout) [reply src/components/LayoutHeader/Header.js]: done

File: src/components/LayoutHeader/Header.js (hunk 4 above)
Comment (fkanout) [reply src/components/LayoutHeader/Header.js]: done

File: src/components/LayoutHeader/Header.js (hunk 5 above)
Comment (fkanout) [reply src/components/LayoutHeader/Header.js]: done

**Comment by fkanout** [comment]
thanks for the review @asantarissy  👍

**Pull request merged**